db/commitlog/commitlog.cc

```diff
@@ -111,6 +111,9 @@ db::commitlog::config db::commitlog::config::from_db_config(const db::config& cf
     if (cfg.commitlog_flush_threshold_in_mb() >= 0) {
         c.commitlog_flush_threshold_in_mb = cfg.commitlog_flush_threshold_in_mb();
     }
+    if (cfg.commitlog_max_data_lifetime_in_seconds() > 0) {
+        c.commitlog_data_max_lifetime_in_seconds = cfg.commitlog_max_data_lifetime_in_seconds();
```

Comment:
Since the target is a plain optional\<integral>, this isn't liveupdatable.

Reply:
See below